### ISSUE TITLE
Migrate styles from CSS modules to CSS-in-JS

### ISSUE THREAD

**Issue by paulcretu** [orcasound/orcahome#28]
Most of the styling is currently being applied from `module.css` files located in [`src/styles`](https://github.com/orcasound/orcahome/tree/master/src/styles). It would be good to rewrite these as CSS-in-JS using the [`sx` prop](https://mui.com/system/the-sx-prop/) and [`styled()`](https://mui.com/system/styled/) that MUI provides. Each file corresponds to a page:

- [ ] About.module.css
- [x] Blog.module.css
- [ ] Home.module.css
- [ ] Layout.module.css
- [ ] Learn.module.css
- [x] Nav.module.css
- [ ] getinvolved.module.css
- [ ] globals.css can probably just be removed and just use [MUI baseline](https://mui.com/components/css-baseline/) instead

The goal should also be to use the [MUI theme](https://mui.com/customization/theming/) as much as possible, and to pull out commonly re-used styles into components. This migration can be done in one swoop, or each page can be refactored one at a time as the latest designs are implemented.

**Comment by YashGupta29** [comment]
Hello,
I would love to contribute. Can you please assign this task to me?

**Comment by Saksham294** [comment]
Hi
I would like to contribute to this issue. Can you please assign it to me?

**Comment by DivyanshuParwal** [comment]
I think refactoring one at a time is a better option as styles will also be changed in those files accordingly with the designs need to be implemented

**Comment by evanjscallan** [comment]
I'd like to contribute to this as well if needed.

**Comment by evanjscallan** [comment]
I've created a reusable theme 'const' using existing stylistic UI attributes for consideration. The theme object would allow the more granular aspects of the CSS styling to be used (mainly with fonts and color palettes) and may speed up the process of refactoring for MUI. Would this be worth submitting as a pull request?

**Comment by DivyanshuParwal** [comment]
@paulcretu Nav.module.css file could be removed from the above list as it is no longer required. Nav styles are already converted to CSS-in-JS in #14 and the file is being removed in #40.

**Comment by paulcretu** [comment]
Thanks @DivyanshuParwal, I've checked it off

@evanjscallan yes, we now have a designated file for theming, [take a look at theme.js](https://github.com/orcasound/orcahome/blob/main/src/styles/theme.ts). You may want to add your changes there and open a PR

**Comment by paulcretu** [comment]
I think @DivyanshuParwal has a good point, it's not worth refactoring the styles if the pages are still being implemented. So let's just not add any new CSS in modules, and removing it when implementing each page. In the meantime I'll try to keep this list updated. Leave a comment if anything gets checked off and I don't notice

**Comment by fspzar123** [comment]
@paulcretu I would like to work on the Layout.module.css issue

**Comment by UXBrendan** [comment]
@paulcretu - does this need to be assigned and completed? If so, how can I help move this item toward completion?

**Comment by paulcretu** [comment]
Yes this still needs to be done. But it's best done as part of the implementation of each page (the other issues). The plan was to reject any PRs that don't move over to CSS-in-JS. So this PR is mostly for tracking the overall progress of the migration.

I don't think it necessarily needs to be assigned right now, but if there are still any CSS modules remaining once the other pages are implemented, then it should be assigned.

**Comment by benjifriedman** [comment]
Resolved in #186
